uberfire-rest/uberfire-rest-client/src/main/java/org/guvnor/rest/client/CreateProjectRequest.java

```diff
@@ -20,18 +20,18 @@
 @Portable
 public class CreateProjectRequest extends JobRequest {
 
-    private String repositoryName;
+    private String ouName;

```

Comment:
Please, use there full name (organizationalUnitName) to make code more readable.